apache-rat-core/src/main/java/org/apache/rat/Report.java

```diff
@@ -246,11 +246,20 @@ public String desc() {
      * @since 0.17.0
      */
     static final Option ARCHIVE = Option.builder().longOpt("archive").hasArg().argName("ProcessingType")
-            .desc(format("Specifies the level of detail in ARCHIVE reporting. (default is %s)",
+            .desc(format("Specifies the level of detail in ARCHIVE file reporting. (default is %s)",
                     ReportConfiguration.Processing.NOTIFICATION))
             .converter(s -> ReportConfiguration.Processing.valueOf(s.toUpperCase()))
             .build();
 
+    /**
+     * Specify the processing of STANDARD files.
+     */
+    static final Option STANDARD = Option.builder().longOpt("standard").hasArg().argName("ProcessingType")
+            .desc(format("Specifies the level of detail in STANDARD file reporting.. (default is %s)",
```

Comment:
Double dot in this sentence - pls update the site template as well to keep it in sync with your changes.